openmetadata-ui/src/main/resources/ui/playwright/utils/team.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { expect, Page } from '@playwright/test';
+import { uuid } from './common';
+
+export const createTeam = async (page: Page, isPublic?: boolean) => {
+  const teamData = {
+    name: `pwteam-${uuid()}`,
+    displayName: `PW ${uuid()}`,
+    email: `pwteam${uuid()}@example.com`,
+    description: 'This is a PW team',
+  };
```

Comment:
instead of this, i would encourage you to use TeamClass for getting dynamic team